src/main/kotlin/org/rust/cargo/project/workspace/CargoWorkspace.kt

```diff
@@ -92,6 +94,17 @@ interface CargoWorkspace {
         LIB, BIN, TEST, EXAMPLE, BENCH, UNKNOWN
     }
 
+    class Feature(
+        val name: String,
+        val state: FeatureState
```

Comment:
Do we really need a state of feature?
Maybe it's enough to keep only enabled features?

Reply:
The initial implementation had just that.
But Rust versions below 1.26 do not give us feature information from `cargo metadata`.
So they currently display Unknown as state.
Not sure if treating all Unknown as Disabled is a good idea.

Reply:
We use 1.26 as the oldest rust version in our tests
Also, the latest rust [survey](https://blog.rust-lang.org/2018/11/27/Rust-survey-2018.html#rust-toolchain) says that almost all users use the latest stable or even nightly build
So, I think we can consider that 1.26 is the minimal supported version by the plugin at this moment